modules/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  config = local.accounts[var.account]
+}
```

Comment:
seems pre-commit hook not worked on your local as we have missing new line at the end of this file